bower.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "chessground",
+  "version": "2.0.11",
+  "description": "lichess.org Chess UI",
+  "main": "chessground.js",
+  "directories": {
+    "example": "examples"
+  },
+  "ignore": [
+    "package.json",
+    "Gulpfile.js",
+    ".gitignore",
+    ".jshintrc",
+    "assets",
+    "screenshot",
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ornicar/chessground.git"
+  },
+  "keywords": [
+    "chess",
+    "lichess.org",
+    "chessboard",
+    "UI"
+  ],
+  "scripts": {
+    "build": "npm install && gulp"
+  },
+  
+  "author": "Thibault Duplessis (https://github.com/ornicar)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ornicar/chessground/issues"
+  },
+  "homepage": "https://github.com/ornicar/chessground"
+}
```

Comment:
I don't know about bower, but I note that this file duplicates a lot of information from `package.json`, which doesn't seem right to me.